src/commands/lambda/flare.ts

```diff
@@ -158,13 +217,13 @@ export class LambdaFlareCommand extends Command {
     if (tagsLength === 0) {
       this.context.stdout.write(commonRenderer.renderSoftWarning(`No resource tags were found.`))
     } else {
-      this.context.stdout.write(`✅ Found ${tagsLength} resource tags.\n`)
+      this.context.stdout.write(`Found ${tagsLength} resource tag(s).\n`)
     }
 
     // Get CloudWatch logs
     let logs: Map<string, OutputLogEvent[]> = new Map()
     if (this.withLogs) {
-      this.context.stdout.write('\n☁️ Getting CloudWatch logs...\n')
+      this.context.stdout.write(chalk.bold('\n🌧  Getting CloudWatch logs...\n'))
```

Comment:
The previous emoji was small on some terminals, so I changed it to this one which is more uniform across terminals